graylog2-web-interface/src/pages/SearchPage.jsx

```diff
@@ -30,6 +30,7 @@ const SearchPage = React.createClass({
     Reflux.connect(CurrentUserStore),
     Reflux.listenTo(InputsStore, '_formatInputs'),
     Reflux.listenTo(RefreshStore, '_setupTimer', '_setupTimer'),
+    Reflux.listenTo(DecoratorsStore, '_refreshDataFromDecoratorStore', '_refreshDataFromDecoratorStore')
```

Comment:
This is the [standard way of 0.2.x reflux](https://github.com/reflux/refluxjs/tree/v0.2.13#using-refluxlistento), unfortunately.